TASK 2

*   total sales revenue
*   total number of customers
*   average number of transactions per customer





In [2]:
import pandas as pd
import matplotlib.pyplot as plt

In [6]:
data=pd.read_csv("/content/QVI_data.csv")

In [7]:
data

LYLTY_CARD_NBR        DATE  ...              LIFESTAGE  PREMIUM_CUSTOMER
0                 1000  2018-10-17  ...  YOUNG SINGLES/COUPLES           Premium
1                 1002  2018-09-16  ...  YOUNG SINGLES/COUPLES        Mainstream
2                 1003  2019-03-07  ...         YOUNG FAMILIES            Budget
3                 1003  2019-03-08  ...         YOUNG FAMILIES            Budget
4                 1004  2018-11-02  ...  OLDER SINGLES/COUPLES        Mainstream
...                ...         ...  ...                    ...               ...
264829         2370701  2018-12-08  ...         YOUNG FAMILIES        Mainstream
264830         2370751  2018-10-01  ...         YOUNG FAMILIES           Premium
264831         2370961  2018-10-24  ...         OLDER FAMILIES            Budget
264832         2370961  2018-10-27  ...         OLDER FAMILIES            Budget
264833         2373711  2018-12-14  ...  YOUNG SINGLES/COUPLES        Mainstream

[264834 rows x 12 columns]

In [25]:
total_sale=data.TOT_SALES.sum()

**TOTAL SALES**

In [9]:
total_sale

1933114.9999999998

In [10]:
data.describe()

LYLTY_CARD_NBR      STORE_NBR  ...      TOT_SALES      PACK_SIZE
count    2.648340e+05  264834.000000  ...  264834.000000  264834.000000
mean     1.355488e+05     135.079423  ...       7.299346     182.425512
std      8.057990e+04      76.784063  ...       2.527241      64.325148
min      1.000000e+03       1.000000  ...       1.500000      70.000000
25%      7.002100e+04      70.000000  ...       5.400000     150.000000
50%      1.303570e+05     130.000000  ...       7.400000     170.000000
75%      2.030940e+05     203.000000  ...       9.200000     175.000000
max      2.373711e+06     272.000000  ...      29.500000     380.000000

[8 rows x 7 columns]

LYLTY_CARD_NBR shows total number of customers i.e 2.648340e+05	

In [24]:
total_customer= data.LYLTY_CARD_NBR.count()
total_customer

264834

In [21]:
total_transaction=data.shape[0] # total_transaction would be no. of rows
total_transaction

264834

**Average number of transactions per customer**

In [22]:
avg=total_transaction/total_customer
avg

1.0

**CONCLUSION** -
* total sales revenue -1933114.9999999998
* total number of customers- 264834
* average number of transactions per customer - 1.0